### DISCUSSION TITLE
install packages di python

### DISCUSSION THREAD

**Discussion by joejnt** [teamalgoritma/community#225] [Q&A] [answered]
Dear team Algoritma,

Saya memiliki masalah untuk membuat connection ke database yg type nya DB2. Saya dapat artikel untuk membuat connection, saya perlu install packages ibm_db atau ibm_db==2.0.8a. Saya melakukan install untuk kedua packages tersebut. Pertama saya melakukan:
- pip install ibm_db (hasilnya sukses) kemudian;
- pip install ibm_db==2.08a Untuk hasil install packages ini saya mendapatkan info seperti ini 
<img width="670" alt="image" src="https://user-images.githubusercontent.com/103427989/181019116-88ff6161-ac4d-4aa2-a20d-952b376254d8.png">

Pertanyaan saya, 
1. apakah proses install packagesnya tidak ada masalah ?
2. Apakah setelah melakukan proses install packages saya perlu melakukan proses ini: python -m ipykernel install --user --name=caats (nama inv yang saya buat)

NB: 
- hasil capture setelah proses install packages dan saya lakukan proses ini `python -m ipykernel install --user --name=caats `
- link article https://stackoverflow.com/questions/6044326/how-to-connect-python-to-db2

Mohon sarannya
Best regars,
JT

**Comment by idoali** [comment, marked as answer]
Bapak bisa coba menggunakan library `ibm_db` yang sudah bapak install terlebih dahulu untuk aplikasi bapak. Jika library tersebut bisa dijalankan maka tidak ada masalah. Untuk `python -m ipykernel install --user --name=caats` sendiri, saya masih belum bisa menemukan tujuan utamanya. Mungkin akan lebih baik jika bapak menginstall library di environment yang sudah biasa bapak pakai jadi tidak perlu menginstall kernel lagi. 

Semoga jawaban ini dapat membantu pak.

**Reply by joejnt** [reply]
@idoali , noted..terima kasih untuk sarannya

**Comment by kevbow** [comment]
Dear @joejnt,

Penggunaan syntax `python -m ipykernel install --user --name=caats` digunakan untuk membuat kernel agar environment bisa digunakan di jupyter notebook. Apabila tidak dipakai maka environment tidak akan terbaca.

**Reply by joejnt** [reply]
Dear @kevbow 

Setelah saya melakukan install `ibm_db` di env, apakah saya perlu menjalankan syntax ini `python -m ipykernel install --user --name=caats` ?
